### PR TITLE
Add Chart.yaml version field updating to release PR automation

### PR DESCRIPTION
**Pull request by ubergesundheit**
This PR adds code to update the version field in Chart.yaml of helm charts

In collaboration with @Gacko 

### Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: pkg/gen/input/workflows/internal/file/create_release_pr.yaml.template [hunk 1]
```diff
@@ -162,4 +175,4 @@ jobs:
           base: "${{ needs.gather_facts.outputs.base }}"
           version: "${{ needs.gather_facts.outputs.version }}"
         run: |
-          hub pull-request -f  -m "Release v${{ env.version }}" -a ${{ github.actor }} -b ${{ env.base }} -h ${{ needs.gather_facts.outputs.branch }}
+          hub pull-request -f -m "Release v${{ env.version }}" -a ${{ github.actor }} -b ${{ env.base }} -h ${{ needs.gather_facts.outputs.branch }}
```
Comment (ubergesundheit) [inline pkg/gen/input/workflows/internal/file/create_release_pr.yaml.template:178]: Unrelated whitespace change

**Review by Gacko — APPROVED** [review]
